lua/astrocommunity/editing-support/nvim-devdocs/README.md

```diff
@@ -0,0 +1,10 @@
+# nvim-devdocs
+
+nvim-devdocs is a plugin which brings [DevDocs][devdocs-site] documentations into neovim.
+Install, search and preview documentations directly inside neovim in markdown format with telescope integration.
+You can also use custom commands like [glow][glow-repo] to render the markdown for a better experience.
+
+_Astrocommunity note: [glow][glow-repo] render will work out of the box if you have it on your PATH_
+
+[glow-repo]: https://github.com/charmbracelet/glow
+[devdocs-site]: https://devdocs.io/
```

Comment:
Today I learnt that this is possible  😯

Reply:
I guess `:DevdocsOpenFloat markdown` would help :D